infection.json.dist

```diff
@@ -0,0 +1,11 @@
+{
+    "timeout": 10,
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "logs": {
+        "text": "infection-log.txt"
+    }
```

Comment:
The new 'default' infection config also includes
```json
   "mutators": {
        "@default": true
    }
```

If no mutators are specified the 'default' profile is used. But if there is configuration done to mutators, then you need to add the `"@default": true` in order to keep the rest of the 'unconfigured' mutators.